### DISCUSSION TITLE
Label not visible in Dark Mode

### DISCUSSION THREAD

**Discussion by yashwanthkrishnab** [zitadel/zitadel#5791] [Q&A] [answered]
<img width="1219" alt="image" src="https://user-images.githubusercontent.com/126572385/235690481-054c4984-578d-479f-a540-383b0074f889.png">
Label is not visible in Dark Mode

**Comment by hifabienne** [comment, marked as answer]
Hei. 
ZITADEL does calculate some of the colors based on the mode you are on. And when you configure a light design in the dark mode this can lead to some strange behaviour.
We already have created the following issue #5505 which should allow you to set which themes you like to have

**Reply by yashwanthkrishnab** [reply]
```zitadel/console/src/styles.scss```
```sass
$caos-dark-app-theme: (
  primary: mat-palette($caos-dark-primary),
  accent: mat-palette($caos-dark-primary),
  warn: mat-palette($caos-dark-warn),
  is-dark: true,
  foreground: $caos-dark-theme-foreground,
  background: $caos-dark-theme-background,
);

$caos-light-app-theme: (
  primary: mat-palette($caos-light-primary),
  accent: mat-palette($caos-light-primary),
  warn: mat-palette($caos-light-warn),
  is-dark: false,
  foreground: $caos-light-theme-foreground,
  background: $caos-light-theme-background,
);
```

By swapping these theme variables can we force light theme @hifabienne
